easybuild/easyconfigs/t/Trinity/Trinity-2.5.1-intel-2017a.eb

```diff
@@ -0,0 +1,41 @@
+name = 'Trinity'
+version = '2.5.1'
+
+homepage = 'http://trinityrnaseq.github.io'
+description = """Trinity represents a novel method for the efficient and robust de novo reconstruction
+ of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
+ Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'optarch': True}
+
+source_urls = ['https://github.com/trinityrnaseq/trinityrnaseq/archive/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+patches = [
+    'chrysalis_commandline_noconsts_2012-10-05.patch',
+    'Trinity-%(version)s_plugins-Makefile.patch',
+    'Trinity-%(version)s_icpc.patch',
+]
+checksums = [
+    'bf14f96981bb028d7960dec895a1bcbda1fadbbe562a913b188559fd74b90457',  # Trinity-v2.5.1.tar.gz
+    # chrysalis_commandline_noconsts_2012-10-05.patch
+    '3442803c357576557de843c80b73436e59e9a99027d60f5f63bb3368eceb4a5c',
+    '2682f182264bf101d0a2b182ca4110f2d1ab16c56e69e804379023c1e1a48847',  # Trinity-2.5.1_plugins-Makefile.patch
+    'a686eafb11d063b8a604190389db98808aaedf3458973b4124c38c9019878901',  # Trinity-2.5.1_icpc.patch
+]
+
+builddependencies = [
+    ('Autotools', '20150215'),
+]
+
+dependencies = [
+    ('Java', '1.8.0_144', '', True),
```

Comment:
Use latest `1.8.0_152`?

You will need to include an `ant` easyconfig that uses it (make that latest `ant` version too).